docs/advanced/addons/pcidevices.md

```diff
@@ -104,6 +104,10 @@ A `USBDevice` resource in Harvester represents a USB device on the node. USB dev
 
 USB passthrough is slightly different from PCI passthrough. For example, you can fully control a USB controller with four USB ports by creating a `PCIDeviceClaim`. However, you can also create a `USBDeviceClaim` to control only one USB port. The other three USB ports remain available to the node.
 
+:::caution
+Please detach the USB device from VM and disable it before unplugging it.
```

Comment:
```suggestion
Before you remove the USB device, detach it from the virtual machine and then disable passthrough on the **USB Devices** screen.
```

Reply:
I used *disable passthrough* because the UI control is `Disable Passthrough` (not Disable Device).

Reply:
Sounds good, thanks!